### ISSUE TITLE
Dalli gem warning about require client.

### ISSUE THREAD

**Issue by raivil** [Shopify/identity_cache#506]
Hey,

I couldn't find a related issue, but `dalli` gem version `3.0.4` (and possible other versions) is printing this alert:
```
You can remove `require 'dalli/cas/client'` as this code has been rolled into the standard 'dalli/client'.
```
The code with the require is https://github.com/Shopify/identity_cache/blob/main/lib/identity_cache/mem_cache_store_cas.rb#L2 (v1.1.0)

**Comment by eugeneius** [comment]
I don't know - I'm not a maintainer, I just wrote the fix.

In the meantime, we added an empty file to our project at `lib/dalli/cas/client.rb` so that the require doesn't trigger the warning.

**Comment by rafaelfranca** [comment]
You can also use the gem in the main branch by pointing your gemfile to it. Releases aren't required anymore in the Bundler era.

**Comment by dylanahsmith** [comment]
I've just released version 1.2.0
